apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.scss

```diff
@@ -151,9 +151,23 @@
     }
 }
 .add-question-footer {
-    margin-top: 40px;
-    margin-bottom: 100px;
+    margin-top: 2.5rem;
+    padding-bottom: 2.5rem;
+    margin-bottom: 2.5rem;
+    width: 120%;
 }
 .text-align-center {
     text-align: center;
+}
+.edit-question-header {
+    display: flex;
+    gap: 1rem;
+    align-self: stretch;
+    span {
+        font-family: Public Sans;
+        font-size: 22px;
```

Comment:
Suggestion: We typically prefer the use of `rem` over `px` for font sizing. 

Also, in the designs there are a lot of places where styling is provided that mimics a `heading` element (h2,h3, etc). This might be one of those. There is a component that can be used to prevent having to define the styling over and over. Adam created the `Heading` component that accepts a `level` prop. This span i believe could be replaced by `<Heading level={2}>...text content</Heading>`